cardano-cli/src/Cardano/CLI/EraBased/Run/CreateTestnetData.hs

```diff
@@ -244,7 +253,17 @@ runGenesisCreateTestNetDataCmd Cmd.GenesisCreateTestNetDataCmdArgs
   g <- Random.getStdGen
 
   -- Distribute M delegates across N pools:
-  delegations <- liftIO $ Lazy.forStateM g distribution $ flip computeInsecureDelegation networkId
+  delegations <-
+    case stakeDelegatorsKind of
+      OnDisk -> do
+        let delegates = concat $ repeat stakeDelegatorsDirs
+            delegatesAndPools = zip delegates distribution
+        -- We don't need to be attentive to laziness here, because anyway this
```

Comment:
What's the context for this comment here?

Reply:
The benchmarking team asked us to stop writing credentials to disk (they were generating massive numbers of keys and I believe it was slowing down their tests). In the case where we want to write keys to disk we basically accept that if we are using large numbers of keys there will be a performance degradation.